module/_build/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Should we be extending a base tsconfig rather than having a build specific one? It makes it easier to split out test code and the like.

Reply:
yeah, potentially! will have a play now to see if I can get that working

Reply:
@judithrocketmakers this better? 
```
{
  "extends" : "../tsconfig.json",
  "compilerOptions": {
    "module": "commonjs",
  }
}
```
This file is just for running my sass concat scripts so hopefully pretty much no impact anyway

Reply:
Looks good :)